packages/kbn-securitysolution-grouping/src/containers/query/types.ts

```diff
@@ -19,11 +19,11 @@ type RunTimeMappings =
   | Record<string, Omit<RuntimeFieldSpec, 'type'> & { type: RuntimePrimitiveTypes }>
   | undefined;
 
-interface BoolAgg {
+export interface BoolAgg {
   bool: BoolQuery;
 }
 
-interface RangeAgg {
+export interface RangeAgg {
```

Comment:
I had to export some types due to the following linter errors:

> exported variable 'getGroupedFindingsQuery' has or is using name 'BoolAgg' from external module "/path/to/query/types" but cannot be named.

> exported variable 'getGroupedFindingsQuery' has or is using name 'RangeAgg' from external module "/path/to/query/types" but cannot be named.

The cloud_security_posture plugin is not directly using those types, but it's using types that consume them, thus confusing the CI typescript linter when inferring these values.